email/parse.c

```diff
@@ -1126,20 +1127,20 @@ size_t mutt_rfc822_read_line(FILE *fp, struct Buffer *buf)
       } while (off && isspace(line[--off]));
 
       /* check to see if the next line is a continuation line */
-      char ch = fgetc(fp);
+      int ch = fgetc(fp);
       if ((ch != ' ') && (ch != '\t'))
       {
         /* next line is a separate header field or EOH */
         ungetc(ch, fp);
         buf_addstr(buf, line);
         break;
       }
-      ++read;
+      read++;
```

Comment:
why?

Reply:
Sorry, that's not coverity related;
it belongs in the flatcap's-needlessly-tidying branch.